src/layouts/DashboardLayout/TopBar.tsx

```diff
@@ -42,6 +62,19 @@ const TopBar = ({
           <Logo />
         </RouterLink>
         <Box flexGrow={1} />
+        <FormControl variant="outlined" className={classes.formControl}>
+          <InputLabel id="select-label">
+          </InputLabel>
+          <Select
+            labelId="select-label"
+            value={locale}
+            onChange={handleChangeLocale}
+          >
+            {localeDatas.map((item, index) => (
+              <MenuItem key={index} value={item}>{item}</MenuItem>
```

Comment:
kluc by som nedaval index, ale to co mas teraz v `item` pretoze to je unique
hodnota by mohla byt uppercase (vo vseobecnosti by sa riesili preklady ale kym nemame ziadne s pomlckou tak to zjednodusime)
Nazov premnnej v lambde by som dal  specifickejsi (pripravene pre buduce rozsirovanie)
navrhujem toto: `<MenuItem key={locale} value={locale}>{locale.toUpperCase()}</MenuItem>`